app/models/item.rb

```diff
@@ -62,6 +62,11 @@ class Item < ApplicationRecord
       .alphabetized
   }
 
+  scope :disposable, -> {
+    joins(:base_item)
+      .where("lower(base_items.category) LIKE '%diaper%'").where.not("lower(base_items.category) LIKE '%cloth%' OR lower(base_items.name) LIKE '%cloth%'")
+  }
+
```

Comment:
This is really clever (not a bad thing).  I like that you consulted the base_items for it! 

If disposability becomes a bigger concern for metrics it might need to be addressed at a more abstract scale (again, not needing to be done in this PR. What you did here looks great!)